### DISCUSSION TITLE
CSSBattle at CSS Café on 16 December

### DISCUSSION THREAD

**Discussion by gunnarbittersmann** [cssbattle/forum#197] [Announcements]
Christian and Nils from [CSS Café](https://www.meetup.com/de-DE/CSS-Cafe/) (great online meet-up) have invited me to talk about CSSBattle. I’m gonna show some magic tricks. 🎩 I hope I’ll be able to stop myself from giving away too many secrets. 😏

Save the date: 2021-12-16 16:00 UTC [CSSBattle at CSS Café](https://www.meetup.com/de-DE/CSS-Cafe/events/282467056/)

**Comment by chinchang** [comment]
wohooO! This is gonna be fun! Go Gunnar! 😃

**Comment by Beowolve** [comment]
Thanks, Gunnar. You definitely surprised me with your song and the last target you have chosen 👍.
![odoo_30](https://user-images.githubusercontent.com/32326640/146420632-66f68e4d-ee4a-4b86-a5f2-cca051be5dbc.gif)

**Reply by chinchang** [reply]
Looks like the meetup helped you learn some new tricks 😄! You are on fire on that target 🔥👏🏽

**Reply by Beowolve** [reply]
Unfortunately no, but gunnar did his best to make the meetup entertaining. How about a new battle to keep us even more entertained? So I don't need to play with keyframes 😄. Yesterday would have been a good day for a new battle to start, but my hopes were not fulfilled.

**Comment by jacobovillegas** [comment]
![image](https://user-images.githubusercontent.com/435754/148041240-3b10fc3c-a1aa-4d75-ad2b-0d12f13f34ad.png)
